app/serializers/insecure_user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class InsecureUserSerializer < ActiveModel::Serializer
+  attributes :id, :email, :password, :name
+
+  def name
```

Comment:
I was following the serializer guide and realized afterwards that the model in the guide (full_name) was actually not needed. I changed it while I finished up but forgot to delete it. 



> On Dec 29, 2015, at 9:09 PM, David Roberts <notifications@github.com> wrote:
> 
> In app/serializers/insecure_user_serializer.rb:
> 
> > @@ -0,0 +1,7 @@
> > +class InsecureUserSerializer < ActiveModel::Serializer
> > +  attributes :id, :email, :password, :name
> > +
> > +  def name
> Why use a method instead of an attribute?
> 
> —
> Reply to this email directly or view it on GitHub.
>